### PR TITLE
Change devise sessions params and route

### PR DESCRIPTION
**Pull request by lucasenriquez**
## Summary

[Change!] 

- [x] I have changed the devise sessions params and routes
- [ ] BEFORE MERGE: I have updated the TRELLO card column before merging the pull request

## TRELLO Card

[Change!] https://trello.com/c/4NJFrdve/26-implementar-autenticaci%C3%B3n

### CODE REVIEW

File: app/controllers/sessions_controller.rb [hunk 1]
```diff
@@ -1,14 +1,17 @@
 class SessionsController < Devise::SessionsController
-    before_action :authenticate_user!
     respond_to :json
-  
-    private
-  
+    before_action :rewrite_param_names, only: [:create]
```
Comment (AccifonteFranco) [inline app/controllers/sessions_controller.rb]: leave an empty line between before_action and create method

File: app/controllers/sessions_controller.rb [hunk 2]
```diff
@@ -1,14 +1,17 @@
...
+    def create
+      super
+    end
```
Comment (AccifonteFranco) [inline app/controllers/sessions_controller.rb]: leave an empty line between the end of a method and the beginning of the next one

File: app/controllers/sessions_controller.rb [hunk 3]
```diff
@@ -1,14 +1,17 @@
...
     def respond_with(resource, _opts = {})
       render json: resource
     end
   
     def respond_to_on_destroy
       head :no_content
     end
+    def rewrite_param_names
+        request.params[:user] = {email: request.params[:session][:email],password: request.params[:session][:password]}
```
Comment (AccifonteFranco) [inline app/controllers/sessions_controller.rb]:
```suggestion
        request.params[:user] = { email: request.params[:session][:email], password: request.params[:session][:password] }
```
The logic is perfect! But rubocop should have warned you about those missing spaces, and that the line is longer than 100 characters.

**Review by AccifonteFranco — CHANGES_REQUESTED** [review]
It's looking good, you only have rubocop offenses. CircleCI should have warned you about them. Check if there are warnings, and if not talk to me.

**Review by dante-wd — APPROVED** [review]

**Pull request merged**